h/static/styles/annotations.scss

```diff
@@ -34,7 +34,14 @@
   margin: .8em 0;
 }
 
-.annotation-header { margin-top: 0 }
+.annotation-header {
+  margin-top: 0;
+
+  &.please-sign-in {
```

Comment:
Sorry, I've been working on other projects the past 2 days. I can't read your mind but I took a stab at removing the extra class. Have a look!